Zombie Defense DTM AD/map.xml

```diff
@@ -1,275 +1,326 @@
+    <union id="zombies-only">
+      <rectangle min="73,-87" max="87,-73"/><!--NE-->
+      <rectangle min="86,73" max="73,87"/><!--SE-->
+      <rectangle min="-87,73" max="-73,87"/><!--SW-->
+      <rectangle min="-87,-73" max="-73,-87"/><!--NW-->
+    </union>
+    <cuboid id="steves-only" min="-7,-oo,-7" max="6,24,6"/>
+    <union id="unbreakable">
+      <!--Spawner column islands-->
+      <!--SE-->
+      <cylinder base="48.5,12,48.5" radius="6" height="3"/>
+      <block>48,15,48</block>
+      <!--SW-->
+      <cylinder base="-48.5,12,48.5" radius="6" height="3"/>
+      <block>-49,15,48</block>
+      <!--NW-->
+      <cylinder base="-48.5,12,-48.5" radius="6" height="3"/>
+      <block>-49,15,-49</block>
+      <!--NE-->
+      <cylinder base="48.5,12,-48.5" radius="6" height="3"/>
+      <block>48,15,-49</block>
+      <!--Orange stained glass pillar-->
+      <cuboid min="4,26,-4" max="5,50,-5"/> 
+      <cuboid min="-5,26,-5" max="-4,50,-4"/>
+      <cuboid min="-5,26,5" max="-4,51,4"/> 
+      <cuboid min="5,26,5" max="4,50,4"/>
+      <!--White stained glass panel-->
+      <cuboid min="-4,46,4" max="4,47,-4"/>
+    </union>
+    <complement id="noteamgrief-z">
+      <cuboid min="-87,-oo,-87" max="87,6,87"/>
+      <rectangle min="-67,-67" max="67,67"/>
+      <rectangle min="67,-52" max="87,-30"/>
+      <rectangle min="67,30" max="87,52"/>
+      <rectangle min="30,67" max="52,87"/>
+      <rectangle min="-52,67" max="-30,87"/>
+      <rectangle min="-87,30" max="-67,52"/>
+      <rectangle min="-87,-51" max="-67,-30"/>
+      <rectangle min="-52,-87" max="-30,-67"/>
+      <rectangle min="30,-87" max="52,-67"/>
+    </complement>
+    <rectangle id="grass area rec" min="-20,-20" max="20,20"/>
+    <complement id="noteamgrief-s">
+      <region id="steves-side"/>
+      <region id="grass area rec"/>
+    </complement>
+    <complement id="wooden deck">
+      <cuboid min="-28,35,-28" max="28,36,28"/>
+      <cuboid min="-21,35,-21" max="21,36,21"/>
+    </complement>
+    <complement id="fort-roof">
+      <cuboid min="-29,27,-29" max="29,28,29"/>
+      <cuboid min="-20,27,-20" max="20,28,20"/>
+    </complement>
+    <union id="zombies-notube">
+      <negative>
+        <complement>
+          <cuboid min="-87,50,-87" max="87,57,87"/>
+          <cuboid min="-67,50,-67" max="67,57,67"/>
+        </complement>
+      </negative>
+    </union>
+    <union id="steves-nohead">
+      <negative>
+        <cuboid min="-8,53,-8" max="8,67,8"/>  
+      </negative>
+    </union>
+    <union id="noexplosion">
+      <cuboid min="-20,-oo,-20" max="20,21,20"/>
+      <region id="noteamgrief-z"/>
+      <region id="wooden deck"/>
+      <region id="fort-roof"/>
+      <cuboid min="-30,20,-30" max="30,21,30"/>
+      <cuboid min="-30,5,-30" max="30,6,30"/>
+    </union>
+    <cuboid id="release" min="-86,50,-86" max="86,51,86"/>
+    
+    <apply block="never" message="Oops! This is uneditable area!" region="uneditable-area"/>
```

Comment:
`Oops! You can't edit this area.` Maybe? 

`This is uneditable area` would be grammatically incorrect